llama/patches/0029-fix-vulkan-PCI-ID-and-ID-handling.patch

```diff
@@ -0,0 +1,236 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Daniel Hiltgen <daniel@ollama.com>
+Date: Fri, 24 Oct 2025 15:21:14 -0700
+Subject: [PATCH] fix vulkan PCI ID and ID handling
+
+---
+ ggml/include/ggml-backend.h          |  3 ---
+ ggml/src/ggml-cuda/ggml-cuda.cu      | 15 +++------------
+ ggml/src/ggml-impl.h                 |  2 +-
+ ggml/src/ggml-vulkan/ggml-vulkan.cpp | 27 +++++++++++++--------------
+ ggml/src/mem_hip.cpp                 | 20 ++++++++++++--------
+ 5 files changed, 29 insertions(+), 38 deletions(-)
+
+diff --git a/ggml/include/ggml-backend.h b/ggml/include/ggml-backend.h
+index 094fc3c82..809835243 100644
+--- a/ggml/include/ggml-backend.h
++++ b/ggml/include/ggml-backend.h
+@@ -174,9 +174,6 @@ extern "C" {
+         int compute_major;
+         int compute_minor;
+         int integrated;
+-        int pci_bus_id;
+-        int pci_device_id;
+-        int pci_domain_id;
+         const char *library;
+         // number with which the devices are accessed (Vulkan)
+         const char *numeric_id;
+diff --git a/ggml/src/ggml-cuda/ggml-cuda.cu b/ggml/src/ggml-cuda/ggml-cuda.cu
+index 816597d2f..b075a18be 100644
+--- a/ggml/src/ggml-cuda/ggml-cuda.cu
++++ b/ggml/src/ggml-cuda/ggml-cuda.cu
+@@ -3504,9 +3504,6 @@ struct ggml_backend_cuda_device_context {
+     int driver_major;
+     int driver_minor;
+     int integrated;
+-    int pciBusID;
```

Comment:
Could this not be removed in the 0026-GPU ... patch so avoiding this and then removing it again